src/Sheet.php

```diff
@@ -349,8 +350,8 @@ public function fromView(FromView $sheetExport)
         /** @var Html $reader */
         $reader = IOFactory::createReader('Html');
 
-        // Insert content into the last sheet
-        $reader->setSheetIndex($spreadsheet->getSheetCount() - 1);
+        // If no sheetIndex given, insert content into the last sheet
+        $reader->setSheetIndex($sheetIndex === null ? $spreadsheet->getSheetCount() - 1 : $sheetIndex);
```

Comment:
`$sheetIndex ?? $spreadsheet->getSheetCount() - 1`

Reply:
I'm afraid not,  if sheetIndex=0, the return is not wright.

Reply:
I think `??` should work. `0 ?? 1 = 0`